certified-connectors/Asite (UAE)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1163 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Asite (UAE)",
+    "description": "Transform how your teams work together with the Asite connector, the ultimate collaboration solution to build connections between business systems.",
```

Comment:
@asitedeveloper Same here

Reply:
@sriyen-msft We have mentioned region in description.